gameplay/src/Matrix.cpp

```diff
@@ -906,12 +906,12 @@ void Matrix::transformPoint(const Vector3& point, Vector3* dst) const
 void Matrix::transformVector(Vector3* vector) const
 {
     GP_ASSERT(vector);
-    transformVector(vector->x, vector->y, vector->z, 0.0f, vector);
+    transformVector(vector->x, vector->y, vector->z, 1.0f, vector);
```

Comment:
I suppose the name of the method is bad. This was supposed to be for transforming a direction vector while transformPoint is for transforming a position. If you transform a direction vector, you don't want it to be affected by scale or position. This is like the difference between  [Transform.TransformDirection](http://docs.unity3d.com/ScriptReference/Transform.TransformDirection.html) and [Transform.TransformPoint](http://docs.unity3d.com/ScriptReference/Transform.TransformPoint.html) in Unity.